test/unit/org/apache/cassandra/cql3/functions/masking/ColumnMaskQueryWithDefaultTest.java

```diff
@@ -39,7 +42,11 @@ public static Collection<Object[]> options()
             options.add(new Object[]{ order, "DEFAULT", "text", "abc", "****" });
             options.add(new Object[]{ order, "DEFAULT", "int", 123, 0 });
             options.add(new Object[]{ order, "mask_default()", "text", "abc", "****" });
-            options.add(new Object[]{ order, "mask_default()", "int", 123, 0, });
+            options.add(new Object[]{ order, "mask_default()", "int", 123, 0 });
+            // TODO: the driver version that we use doesn't support vectors, so we have to use raw values by now
+            options.add(new Object[]{ order, "mask_default()", "vector<int, 2>",
```

Comment:
Is it necessary to add any other element type's case  like float ?

Reply:
These tests focus on how different queries interact with masking rather than on what each particular function and data type combination does. It's useful to test different functions just in case, but we cannot add too many combinations here because each function/type variation multiplies the number of parameterized cases, making the test slower. That's why we only test with a single function and a few data types. At the moment the test takes 2-3 minutes to run, and adding more cases might make it timeout on CI.

Instead, I have added a specific randomized/fuzz test on `AbstractTypeTest` that would cover many data type combinations.